helix-rest/src/main/java/org/apache/helix/rest/metadatastore/accessor/ZkRoutingDataWriter.java

```diff
@@ -77,8 +106,11 @@ public synchronized boolean addMetadataStoreRealm(String realm) {
       return createZkRealm(realm);
     }
 
-    // TODO: Forward the request to leader
-    return true;
+    String urlSuffix =
+        constructUrlSuffix(MetadataStoreRoutingConstants.MSDS_GET_ALL_REALMS_ENDPOINT, realm);
+    return forwardRequestToLeader(urlSuffix,
+        MetadataStoreRoutingConstants.HttpRequestForwardingVerbs.PUT, "addMetadataStoreRealm",
```

Comment:
1. If you want these HTTP verbs included in `MetadataStoreRoutingConstants `, you could use a more generalized name like `HttpVerbs`.  `HttpRequestForwardingVerbs` if you want to keep it as an enum in this class.
2. If you want to generalize and put this in MetadataStoreRoutingConstants, then I'm pretty sure you could already find the supported HTTP verb enum somewhere defined in helix-rest module. Use that instead.

Reply:
Created `HttpConstants.RestVerbs` in `org.apache.helix.rest.common` that serves the enum purpose.